### ISSUE TITLE
[SUGGESTION] Dynamic Height Based Clouds

### ISSUE THREAD

**Issue by Jerzerak** [OreCruncher/DynamicSurroundings#43]
Not sure if this is an entirely feasible/doable addition but cool nonetheless!
https://www.reddit.com/r/minecraftsuggestions/comments/5wft28/lets_take_another_look_at_this_amazing_idea_for/

**Comment by OreCruncher** [comment]
I haven't looked myself, but have you checked out [Clouds Pack](http://www.minecraftforum.net/forums/mapping-and-modding/resource-packs/2339463-extremely-realistic-clouds-pack-60-000-downloads)?

**Comment by Jerzerak** [comment]
I haven't, thanks for the link!

**Comment by OreCruncher** [comment]
@Jerzerak Curious if you checked out that pack.  I been kicking around and it isn't too bad.

**Comment by Jerzerak** [comment]
Yeah I checked it out, looks pretty good! Still leaves something left to be desired but not quite sure what that is yet.
